### ISSUE TITLE
Missing Last Accounts Type - Part 2

### ISSUE THREAD

**Issue by MattHemus** [kevbite/CompaniesHouse.NET#134]
Hi,

I've been doing quite a bit of work with Companies House data recently and I've found another missing Accounts Type.

Could you add the following Last Accounts Type to the enum;

audited-abridged

An example company that returns this is BELMONT CEDAR PARK LIMITED, 07477470.

The missing enum is currently causing an exception when requesting this company.

Many Thanks Again

Matt

**Comment by kevbite** [comment]
Hey @MattHemus,

I'll try to get this in within the next few days, I'll also check if there is any more outstanding from the documentation, however, the documentation always seems out of date compared to what is returned from the API.

**Comment by kevbite** [comment]
Looking at the docs they still don't seem to include these new enumerations 😢.
https://github.com/companieshouse/api-enumerations/blob/master/constants.yml#L136

```yml
account_type:
    'null' : "Null"
    'full' : "Full"
    'small' : "Small"
    'medium' : "Medium"
    'group' : "Group"
    'dormant' : "Dormant"
    'interim' : "Interim"
    'initial' : "Initial"
    'total-exemption-full' : "Total Exemption Full"
    'total-exemption-small' : "Total Exemption Small"
    'partial-exemption' : "Partial Exemption"
    'audit-exemption-subsidiary' : "Audit Exemption Subsidiary"
    'filing-exemption-subsidiary' : "Filing Exemption Subsidiary"
    'micro-entity' : "Micro Entity"
```

**Comment by MattHemus** [comment]
That really doesn't help much does it!

I've downloaded a copy of the latest Companies House data in csv format from the website and querying this it does appear that the only 2 values that you were missing are "unaudited-abridged" and "audited-abridged".

You've obviously already added the first one so hopefully the second one will be the last of them, until they decide to make up some more :-)

Thanks for looking into this.

**Comment by kevbite** [comment]
Fix is now pushed in [7.8.4](https://www.nuget.org/packages/CompaniesHouse/7.8.4).

Please raise anymore that you come across 👍

**Comment by MattHemus** [comment]
Great, thanks for the quick response again.
